gpt_engineer/applications/cli/cli_step_bundle.py

```diff
@@ -0,0 +1,28 @@
+from gpt_engineer.core.ai import AI
+from gpt_engineer.core.code import Code
+from gpt_engineer.data.file_repository import FileRepository
+from gpt_engineer.core.step_bundle_interface import StepBundleInterface
+from gpt_engineer.core.default.steps import gen_code, gen_entrypoint, execute_entrypoint
+from gpt_engineer.core.default.paths import memory_path
+from gpt_engineer.applications.cli.learning import human_review
+
+
+class CliStepBundle(StepBundleInterface):
+
+    def __init__(self, workspace_path: str):
+        self.workspace_path = workspace_path
+        self.memory = FileRepository(memory_path(workspace_path))
+
+
+    def gen_code(self, ai: AI, prompt: str) -> Code:
+        code = gen_code(self.workspace_path, ai, prompt, self.memory)
+        #TODO: evaluate whether it makes more sense to send the code than the memory to gen_entrypoint
+        entrypoint = gen_entrypoint(self.workspace_path, ai, self.memory)
```

Comment:
Code is more specific so should def send that one imo

Reply:
Feels neater! The open question to me is whether there is additional information in the non-code parts of the output which makes it easier for the LLM to understand what the entrypoint should run. I'll change it for now and keep this in mind if it starts under performing.